grpc/src/test/java/com/linecorp/armeria/client/grpc/GrpcClientTest.java

```diff
@@ -394,6 +395,14 @@ void grpcCallOptions() {
         }
     }
 
+    void grpcMethodDescriptor() {
```

Comment:
nit; Will this test actually run?

```suggestion
    @Test
    void grpcMethodDescriptor() {
```

I assume the previous `grpcCallOptions` also needs a `@Test` annotation